src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/IndexPage.java

```diff
@@ -56,4 +66,5 @@ public Object presenceOfReloadButton() {
             return false;
         }
     }
```

Comment:
This can be replaced by the ExpectedConditions i refered to before. I will change this in my code aswell :)